### PR TITLE
add napari-plugin-manager

### PR DESCRIPTION
**Pull request by jaimergp**
Checklist
- [X] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
- [X] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/5eddbd7fc9d1502169089da06c3688d9759be978/recipes/example/meta.yaml#L64-L73) for an example).
- [X] Source is from official source.
- [X] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged).
- [ ] If static libraries are linked in, the license of the static library is packaged.
- [X] Package does not ship static libraries. If static libraries are needed, [follow CFEP-18](https://github.com/conda-forge/cfep/blob/main/cfep-18.md).
- [X] Build number is 0.
- [X] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details).
- [ ] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.
- [X] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

### CODE REVIEW

File: recipes/napari-plugin-manager/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+about:
+  home: https://github.com/napari/napari-plugin-manager
+  summary: Install plugins for napari, in napari.
+  license: BSD-3-Clause
+  license_file: LICENSE
+
+extra:
+  recipe-maintainers:
+    - jaimergp
+    - goanpeca
```
Comment (jakirkham) [inline recipes/napari-plugin-manager/meta.yaml:49]: @goanpeca , are you ok being a maintainer here?

Comment (goanpeca) [reply recipes/napari-plugin-manager/meta.yaml:49]: Yes I m ok 👍🏼

File: recipes/napari-plugin-manager/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,49 @@
+{% set name = "napari-plugin-manager" %}
+{% set version = "0.1.0a2" %}
```
Comment (jakirkham) [inline recipes/napari-plugin-manager/meta.yaml:2]: Just a note this is ok to do as there has [never been a non-alpha release]( https://pypi.org/project/napari-plugin-manager/#history )

Whenever there is a formal release, new alphas, etc. will then need to go under the appropriate label

Here's [the relevant text from CFEP 5]( https://github.com/conda-forge/cfep/blob/31efda3f858fe3574db0418b9d4b4b907159f6f0/cfep-05.md?plain=1#L93-L101 ):

> ## Exceptions
>
> Certain packages (for example [black](https://pypi.org/project/black/#history)) follow
a release cycle in which they have never had a non-beta/alpha release.  In these cases
the conda packages for those do *not* need to be published to a prerelease label.
>
> Once a non-prerelease version of such a package is available and has been merged into
the associated feedstock, they lose this exception and should publish *future* prereleases
to a prerelease label like other packages.

**Comment by jakirkham** [comment]
Thanks Jaime & Gonzalo! 🙏

**Pull request merged**